dspace-spring-rest/src/test/java/org/dspace/app/rest/builder/WorkspaceItemBuilder.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.builder;
+
+import org.dspace.content.Collection;
+import org.dspace.content.Item;
+import org.dspace.content.WorkspaceItem;
+import org.dspace.content.service.DSpaceObjectService;
+import org.dspace.core.Context;
+import org.dspace.eperson.Group;
+
+/**
+ * Builder to construct WorkspaceItem objects
+ *
+ **/
+public class WorkspaceItemBuilder extends AbstractDSpaceObjectBuilder<Item> {
```

Comment:
This should extends AbstractDSpaceObjectBuilder<WorkspaceItem>

Reply:
You are right. So you should start from a lower level in the hierarchy org.dspace.app.rest.builder.AbstractCRUDBuilder<T> or org.dspace.app.rest.builder.AbstractBuilder<T, S>